### PR TITLE
fix(react-grid): add lost border to bands

### PR DESCRIPTION
**Pull request by MaximKudriavtsev**

### CODE REVIEW

File: packages/dx-grid-core/src/plugins/table-band-header/helpers.js [hunk 1]
```diff
@@ -3,10 +3,18 @@ import {
...
 import { TABLE_DATA_TYPE } from '../table/constants';
 import { TABLE_HEADING_TYPE } from '../table-header-row/constants';
+import { TABLE_EDIT_COMMAND_TYPE } from '../table-edit-column/constants';
+import { TABLE_DETAIL_TYPE } from '../table-row-detail/constants';
+import { TABLE_SELECT_TYPE } from '../table-selection/constants';
 
 export const isBandedTableRow = tableRow => (tableRow.type === TABLE_BAND_TYPE);
 export const isBandedOrHeaderRow = tableRow => isBandedTableRow(tableRow)
 || tableRow.type === TABLE_HEADING_TYPE;
+export const isCommandColumn = columnType => (
```
Comment (kvet) [inline packages/dx-grid-core/src/plugins/table-band-header/helpers.js]: Can we ether check that it is not data cell?

File: packages/dx-grid-core/src/plugins/table-band-header/helpers.test.js [hunk 2]
```diff
@@ -397,5 +413,76 @@ describe('TableBandHeader Plugin helpers', () => {
           },
         });
     });
+
+    describe('with command button', () => {
+      const testTableHeaderRows = [
+        { type: 'select' },
```
[the remark this answers is not in the document]
Comment (MaximKudriavtsev) [reply packages/dx-grid-core/src/plugins/table-band-header/helpers.test.js]: Done

[the remark this answers is not in the document]
Comment (MaximKudriavtsev) [reply packages/dx-grid-core/src/plugins/table-band-header/helpers.test.js]: @SergeyAlexeev I don't want to use real type. Can we use abstract type like `commandColumn`?

[the remark this answers is not in the document]
Comment (SergeyAlexeev) [reply packages/dx-grid-core/src/plugins/table-band-header/helpers.test.js]: > I don't want to use real type.

What about `TABLE_BAND_TYPE`, `TABLE_HEADING_TYPE` ?))

File: packages/dx-react-grid-bootstrap4/src/templates/table-band-header/cell.jsx [hunk 3]
```diff
@@ -3,12 +3,15 @@ import * as PropTypes from 'prop-types';
 import classNames from 'classnames';
 
 export const Cell = ({
-  column, children,
+  column, children, leftBorder,
```
Comment (SergeyAlexeev) [inline packages/dx-react-grid-bootstrap4/src/templates/table-band-header/cell.jsx]: What about docs for the `leftBorder` prop?

Comment (MaximKudriavtsev) [reply packages/dx-react-grid-bootstrap4/src/templates/table-band-header/cell.jsx]: Done

File: packages/dx-react-grid/docs/reference/table-band-header.md [hunk 4]
```diff
@@ -46,6 +46,7 @@ Extends [Table.CellProps](table.md#tablecellprops)
 Name | Type | Description
 -----|------------|------------
 children? | ReactNode | A React node used to render band cell content.
+leftBorder? | boolean | Specifies the presence of the left border.
```
[the remark this answers is not in the document]
Comment (viterobk) [reply packages/dx-react-grid/docs/reference/table-band-header.md]: May be beforeBorder?

**Pull request merged**